langlib/langlib-test/src/test/resources/test-src/stringlib_test.bal

```diff
@@ -91,3 +91,7 @@ function testToCodepointInts(string st) returns int[] {
 function testFromCodePointInts(int[] ints) returns string|error {
     return strings:fromCodePointInts(ints);
 }
+
+function testSubstringOutRange() returns string {
+    return "abcdef".substring(7, 9);
+}
```

Comment:
Shall we add a negative test case by giving invalid numbers?  (-1, 5), (5, 3), etc.

Reply:
Added, found and bug and fixed 😃